cognite/neat/workflows/steps/lib/graphs.py

```diff
@@ -0,0 +1,159 @@
+import logging
+from pathlib import Path
+from typing import Tuple
+from cognite.neat.constants import PREFIXES
+
+from cognite.neat.graph.stores import RdfStoreType
+from cognite.neat.graph.stores import NeatGraphStore, drop_graph_store
+from cognite.neat.workflows.model import FlowMessage, WorkflowConfigItem, WorkflowConfigs
+from cognite.neat.workflows.steps.step_model import Step
+
+from ..data_contracts import RulesData, SolutionGraph, SourceGraph
+
+__all__ = ["ConfigureDefaultGraphStores", "LoadInstancesFromRdfFileToSourceGraph", "ResetGraphStores"]
+
+
+class ConfigureDefaultGraphStores(Step):
+    description = "The step initializes the source and solution graph stores."
+    category = "graph_store"
+    configuration_templates = [
+        WorkflowConfigItem(
+            name="source_rdf_store.type",
+            value=RdfStoreType.OXIGRAPH,
+            label="Data store type for source graph. Supported: oxigraph, memory,file, graphdb, sparql. ",
+        ),
+        WorkflowConfigItem(
+            name="solution_rdf_store.type",
+            value=RdfStoreType.OXIGRAPH,
+            label="Data store type for solutioin graph. Supported: oxigraph, memory,file, graphdb, sparql",
+        ),
+        WorkflowConfigItem(
+            name="source_rdf_store.disk_store_dir",
+            value="source-graph-store",
+            label="Local directory for source graph store",
+        ),
+        WorkflowConfigItem(
+            name="solution_rdf_store.disk_store_dir",
+            value="solution-graph-store",
+            label="Local directory for solution graph store",
+        ),
+        WorkflowConfigItem(
+            name="stores_to_configure",
+            value="all",
+            label="Defines which stores to configure. Possible values: all, source, solution",
+        ),
+        WorkflowConfigItem(
+            name="solution_rdf_store.api_root_url",
+            value="",
+            label="Root url for graphdb or sparql endpoint",
+        ),
+    ]
+
+    def run(self, configs: WorkflowConfigs, rules_data: RulesData) -> Tuple[FlowMessage, SourceGraph, SolutionGraph]:
```

Comment:
Incorrect return type. You have 5 different returns, and only one complies with this.